sdk/core/azure-core/CMakeLists.txt

```diff
@@ -32,17 +32,6 @@ if(BUILD_TRANSPORT_CURL)
   message("Libcurl version ${CURL_VERSION_STRING}")
 endif()
 
-# Storage requires 3.6.0 version for using `contains` feature
-set(NLOHMANN_JSON_MIN_REQUIRED_VERSION 3.6.0)
```

Comment:
Add a comment in source somewhere or persist some other indication which lets us remember that we we picked version `3.6.0` of nlohmann json as the source copy, so we know what to update from/to if need be in the future.

Reply:
the version is in the json.hpp file